src/rhsm/config.py

```diff
@@ -15,6 +15,7 @@
 # in this software or its documentation.
 #
 
+from distutils import util
```

Comment:
When you try to import this:
```python
from distutils import util
```

Then following warning is printed: 

```
The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives
  from distutils import util
```

Please provide something that is not deprecated.